### ISSUE TITLE
Binning mpg produces undefined in legend

### ISSUE THREAD

**Issue by domoritz** [vega/vega-lite#961]
Potentially this is an issue with the data itself because the circles all have the same size. 

```json
{
  "data": {
    "url": "data/cars.json"
  },
  "mark": "point",
  "encoding": {
    "x": {
      "bin": true,
      "field": "Displacement",
      "type": "quantitative"
    },
    "size": {
      "bin": true,
      "field": "Miles_per_Gallon",
      "type": "quantitative"
    },
    "y": {
      "field": "*",
      "aggregate": "count",
      "type": "quantitative",
      "displayName": "Number of Records"
    }
  }
}
```

<img width="412" alt="screen shot 2016-01-17 at 17 10 44" src="https://cloud.githubusercontent.com/assets/589034/12381357/65fa2a0c-bd3d-11e5-82c8-32eb26b0d0d2.png">

**Comment by kanitw** [comment]
I don’t think it’s the data itself.  Swapping `y` and `size` works perfectly fine 

```js
{
  "data": {
    "url": "data/cars.json"
  },
  "mark": "point",
  "encoding": {
    "x": {
      "bin": true,
      "field": "Displacement",
      "type": "quantitative"
    },
    "y": {
      "bin": true,
      "field": "Miles_per_Gallon",
      "type": "quantitative"
    },
    "size": {
      "field": "*",
      "aggregate": "count",
      "type": "quantitative",
      "displayName": "Number of Records"
    }
  }
}
```

The real issue is that `bin_Miles_per_Gallon_range` is not included in the `aggregate` transform and thus not included as the output in `summary` table.

**Comment by kanitw** [comment]
Fixed in #968
